diff --git a/inst/aminolookup.m b/inst/aminolookup.m
index 3fe8e49..569ea51 100644
--- a/inst/aminolookup.m
+++ b/inst/aminolookup.m
@@ -116,7 +116,8 @@ function result = aminolookup (varargin)
       %% this is an extension of the matlab options
       searchtype = 'integer';
     elseif ischar (varargin{1})
-      if ((mod (numel (varargin{1}), 3) == 0) && (find (upper (varargin{1})) == 1:3:numel (varargin{1})))
+      nel = numel (varargin{1});
+      if ((mod (nel, 3) == 0) && all(find(isstrprop(varargin{1},'upper')) == [1:3:nel]))
         %% if the number of characters is divisible by 3 and exactly
         %% every third character is upper case
         searchtype = 'abbreviation';
